### PR TITLE
chore(docs): install GWAPI from experimental channel

### PR DESCRIPTION
**Pull request by tao12345666333**
**What this PR does / why we need it**:
Because the standard channel only includes a subset of resources, users who follow this document will not be able to access all the functions that KIC has already provided.

**Which issue this PR fixes**:

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] ~the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR~

### CODE REVIEW

**Review by programmer04 — DISMISSED** [review]
👍 Nice catch. Would you mind checking (and fixing) in https://docs.konghq.com whether we haven't made the same mistake?

**Review by mlavacca — COMMENTED** [review]
Good catch! I think we should give the user the choice on which channel to install.
Something like:
```
If you plane to use alpha resources, such as UDPRoutes, TCRoutes, ...
then <install experimental>
else <install standard>
```
Even considering that the experimental resources are not reconciled unless the `gatewayAlpha` FG is set

**Comment by pmalek** [comment]
> Good catch! I think we should give the user the choice on which channel to install. Something like:
> 
> ```
> If you plane to use alpha resources, such as UDPRoutes, TCRoutes, ...
> then <install experimental>
> else <install standard>
> ```
> 
> Even considering that the experimental resources are not reconciled unless the `gatewayAlpha` FG is set

Probably the best thing to do would be to add a note in quote block referring said FG

**Comment by tao12345666333** [comment]
I have checked and found the following content on the document site. It can be displayed correctly according to the parameters we set for installation from that channel.

https://github.com/Kong/docs.konghq.com/blob/7b9a8329bac080e35866eea68ae15c034410f325/app/_includes/md/kic/prerequisites.md?plain=1#L15-L28

For example: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/services/tcp/#install-the-gateway-apis

File: README.md [hunk 1]
```diff
@@ -39,12 +39,25 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
...
+This command will install all resources that have graduated to beta,
+including GatewayClass, Gateway, ReferenceGrant, and HTTPRoute.
 
 ```shell
-# Install the Gateway API CRDs before installing Kong Ingress Controller.
 kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
-# Install the Kong Ingress Controller with Helm.
+```
+
+Or, if you want to try some experimental resources and fields such as TCPRoute and UDPRoute,
```
Comment (mlavacca) [inline README.md:51]:
```suggestion
Or, if you want to use experimental resources and fields such as `TCPRoute`s and `UDPRoute`s,
```

File: README.md [hunk 2]
```diff
@@ -39,12 +39,25 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
 You can use [Minikube or Kind][k8s-io-tools] on your local machine or use
 a hosted Kubernetes service like [GKE](https://cloud.google.com/kubernetes-engine/).
 
-Setting up Kong for Kubernetes is as simple as:
+### Install the Gateway API CRDs
+
+This command will install all resources that have graduated to beta,
+including GatewayClass, Gateway, ReferenceGrant, and HTTPRoute.
```
Comment (mlavacca) [inline README.md:45]:
```suggestion
This command will install all the GA resources,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Comment (tao12345666333) [reply README.md:45]: Thanks! But the `ReferenceGrant` still in v1beta1
https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/apis/v1beta1/referencegrant_types.go#L43

Comment (tao12345666333) [reply README.md:45]:
```suggestion
This command will install all resources that have graduated to beta,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Comment (tao12345666333) [reply README.md]: I referred to the description here https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/site-src/guides/index.md?plain=1#L38-L40

Comment (randmonkey) [reply README.md]: How about "Resources graduated to GA or beta" here?

Comment (mlavacca) [reply README.md]: It looks like that upstream description is a bit out of date. At the moment, Gateway, GatewayClass and HTTPRoute are v1, while ReferenceGrant is beta. +1 on Tao's suggestion.

Comment (mlavacca) [reply README.md]: https://github.com/kubernetes-sigs/gateway-api/pull/2606

**Review by czeslavo — APPROVED** [review]

**Pull request merged**